src/components/Match/matchColumns.jsx

```diff
@@ -541,11 +528,7 @@ export const inflictorsColumns = [{
     field: 'damage_inflictor',
     displayFn: (row, col, field) => (field ? Object.keys(field)
       .sort((a, b) => field[b] - field[a])
-      .map((k, i) => inflictorWithValue({
-        inflictor: k,
-        value: abbreviateNumber(field[k]),
-        key: i,
-      })) : ''),
+      .map(k => inflictorWithValue(k, abbreviateNumber(field[k]))) : ''),
```

Comment:
why did we switch to non-destructured inputs?  I think it's easier to read if we can have named parameters in the invocations.